Governance/tech/title.md

```diff
@@ -1,10 +1,15 @@
 # Descriptive title
 
-That you are publishing your data or code it is important to have a good title for your new record. Like you probably carefully choose a title for your papers, so you should try to put some effort in creating a good title for othe research outputs. The title will be in any reference to your data or code.  While, depending on the repository you are using to publish, you can potentially change the title, even after a record has been published, it is important to get it right and be as descriptive as possible.
+When publishing data or code it is important to have a descriptive title for the new record. Paper titles are carefully chosen, the same effort should go in creating an effective title for other research outputs, as the title will be in any reference to the data or code. Depending on the repository used to publish, it might be possible to change the title even after a record has been published, however this is best avoided.
 
-Also most people querying a repository will skim quickly through the list of returned records. All they will see initially is the title and the start of the description, until they actually click on one specific record.
+Most people querying a repository will skim quickly through the list of returned records. All they will see initially is the title and the start of the description. Also, any word in the title is used as a keyword by free text search engine.
 
-And finally any word in the title is used as a keyword by free text search engine.
+## What to include
+Keeping in mind that the title words are used as keywords, a good rule of thumb is to include the sort of information that a potential user might choose for a search. The most important are probably the kind of data, the region, temporal coverage and frequency of the data. 
+Where it is relevant the source of the data should also be included, for example if this is a dataset derived from CMIP6, it might be interesting for anyone searching for CMIP6 data. If it is a model output, the name and version of the model would give a lot of useful information to understand the potential use of the data.
+If publishing data or code underlining a paper publication usually this is clearly stated in the title. Similarly, it can be useful to mention the project associated to the research output, as giving the context in which the data or code was created can also clarify potential future use for them.
```

Comment:
who is "them"?
I guess also it helps projects track their research outputs beyond publications.

Reply:
I'll remove "for them" in my mind "them" were the potential users :-)